sql-insert/insert-actor.sql

```diff
@@ -0,0 +1,3 @@
+insert into actors ("firstName", "lastName")
+values ('Tom', 'Hardy');
+SELECT * from actors where "lastName" = 'Hardy';
```

Comment:
It's best to use the `returning` syntax since the SELECT statement here is not guaranteed to return the value that was just inserted (there may be other actors with last name 'Hardy'.)

Same below.